app/controllers/passwords_controller.rb

```diff
@@ -23,7 +23,7 @@ def update
 
     if @user.update_password password_from_password_reset_params
       @user.reset_api_key! if reset_params[:reset_api_key] == "true"
-      @user.api_keys.delete_all if reset_params[:reset_api_keys] == "true"
+      @user.api_keys.expire_all! if reset_params[:reset_api_keys] == "true"
```

Comment:
why not update the `soft_deleted_at` vs `expires_at`? what is the difference between the two (`soft_deleted_at` is used when the user doesn't own the gem that is scoped to a key IIRC) . Could we consolidate them into one concept/attribute since they both convey that the api key cannot be used anymore.

Reply:
because soft_deleted_at is specifically associated with the associated gem no longer being owned by the user, and I don't want to change that meaning.

Also, right now we only check for presence of soft_deleted_at, and not whether its in the past or future